src/hooks/useSelectChain.ts

```diff
@@ -25,7 +25,12 @@ export default function useSelectChain() {
         console.error('Failed to switch networks', error)
 
         dispatch(updateConnectionError({ connectionType, error: error.message }))
-        dispatch(addPopup({ content: { failedSwitchNetwork: targetChain }, key: `failed-network-switch` }))
+        dispatch(
+          addPopup({
+            content: { failedSwitchNetwork: targetChain },
+            key: `failed-network-switch`,
```

Comment:
are these just raw strings and not enums? 
```suggestion
            key: "failed-network-switch",
```